src/components/refactor/dialogs/lines-attach-to-split-lines/lines-attach-to-split-lines-dialog.js

```diff
@@ -0,0 +1,164 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { useSnackMessage } from '@gridsuite/commons-ui';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { sanitizeString } from 'components/dialogs/dialogUtils';
+import PropTypes from 'prop-types';
+import {
+    ATTACHED_LINE_ID,
+    BUS_BAR_SECTION_ID,
+    REPLACING_LINE_1_ID,
+    REPLACING_LINE_2_ID,
+    REPLACING_LINE_1_NAME,
+    REPLACING_LINE_2_NAME,
+    VOLTAGE_LEVEL_ID,
+    LINE_TO_ATTACH_TO_1_ID,
+    LINE_TO_ATTACH_TO_2_ID,
+    CONNECTIVITY,
+    VOLTAGE_LEVEL,
+    ID,
+    BUS_OR_BUSBAR_SECTION,
+} from 'components/refactor/utils/field-constants';
+import yup from 'components/refactor/utils/yup-config';
+import React, { useCallback, useEffect } from 'react';
+import { FormProvider, useForm } from 'react-hook-form';
+import { linesAttachToSplitLines } from 'utils/rest-api';
+import ModificationDialog from 'components/refactor/dialogs/commons/modificationDialog';
+
+import LinesAttachToSplitLinesForm from './lines-attach-to-split-lines-form';
+import {
+    getConnectivityEmptyFormData,
+    getConnectivityFormData,
+    getConnectivityFormValidationSchema,
+} from '../connectivity/connectivity-form-utils';
+
+const emptyFormData = {
+    [LINE_TO_ATTACH_TO_1_ID]: null,
+    [LINE_TO_ATTACH_TO_2_ID]: null,
+    [ATTACHED_LINE_ID]: null,
+    [REPLACING_LINE_1_ID]: null,
+    [REPLACING_LINE_1_NAME]: '',
+    [REPLACING_LINE_2_ID]: null,
+    [REPLACING_LINE_2_NAME]: '',
+    ...getConnectivityEmptyFormData(),
```

Comment:
add a param "withDirectionsInfos" to prevent all unused field to be set in this form